engine.go

```diff
@@ -59,6 +59,12 @@ var defaultOpenAPIConfig = OpenAPIConfig{
 	JSONFilePath: "doc/openapi.json",
 }
 
+// WithRequestContentType sets the accepted content types for the engine.
+// By default, the accepted content types is */*.
+func WithRequestContentType(consumes ...string) func(*Engine) {
```

Comment:
What about naming it


```suggestion
func WithRequestAcceptedContentType(consumes ...string) func(*Engine) {
```


```suggestion
func WithAcceptedContentType(consumes ...string) func(*Engine) {
```

It could also be a plural

```suggestion
func WithAcceptedContentTypes(consumes ...string) func(*Engine) {
```

Here it's unclear. It seems it set the HTTP request Content

You are moving it in a breaking change, renaming is then possible.

Also why is the arguments being called `consumes` it's strange to me

Reply:
`consumes` is a vernacular used by `kin-openapi`

```golang
func NewContentWithSchemaRef(schema *SchemaRef, consumes []string) Content {
	if len(consumes) == 0 {
		return Content{
			"*/*": NewMediaType().WithSchemaRef(schema),
		}
	}
	content := make(map[string]*MediaType, len(consumes))
	for _, mediaType := range consumes {
		content[mediaType] = NewMediaType().WithSchemaRef(schema)
	}
	return content
}
```

https://github.com/getkin/kin-openapi/blob/v0.128.0/openapi3/content.go#L29

Reply:
I don't really want to change the name either had we have a Route Option with a similar name. 

```
// RequestContentType sets the accepted content types for the route.
// By default, the accepted content types is */*.
// This will override any options set at the server level.
func OptionRequestContentType(consumes ...string) func(*BaseRoute) {
	return func(r *BaseRoute) {
		r.AcceptedContentTypes = consumes
	}
}
```

Reply:
If the name was to be changed. It should all be addressed in a the same PR changing the name across the board. Not simply because this function is being moved. The spirit of this change is to consolidate all our server option into an engine option much like my last PRs.